examples/data-sources/kentik-cloudexport_item/run.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+# The script applies the example Terraform configuration.
+# The provider uses stub Kentik API server by default.
+# Production Kentik API server can be used by passing "production" positional argument to the script.
+# TODO(dfurman): deduplicate with run.sh functions in other examples
+
+SCRIPT_DIR=$(dirname "${BASH_SOURCE[0]}")
+REPO_DIR=$(cd -- "$SCRIPT_DIR" && cd ../../../ && pwd)
```

Comment:
why the double cd...?

Reply:
This seems to be a Codilime pattern ... and it seems to work OK. There are other ways to achieve the same thing, but this is fine and reasonable portable and reliable.

Reply:
I have written this one-liner several months ago. I don't remember why, but I suppose it looked more readable to me. I can change it if you insist.

Reply:
It is fine as is.

Reply:
ok, this is replacement for `readlink -f "$SCRIPT_DIR/../../.."`, I see now